tests/unit/sprite/test_sprite.py

```diff
@@ -324,12 +324,10 @@ def test_visible():
     # Make invisible
```

Comment:
If we're changing the behavior, we should explicitly test for the new state persistence.
```suggestion
    # Set an alpha value which the visible property shouldn't change
    sprite.alpha = 127

    # Make invisible
```